back/tests/WebSpec.hs

```diff
@@ -25,23 +26,92 @@ import Control.Monad.Catch
 
 -- Testing
 import Selenium
+import qualified Api
 import qualified Test.WebDriver.Common.Keys as Key
 
 -- Site
 import qualified Guide.Main
-import Guide.Config (Config(..))
+import Guide.Logger
+import Guide.Config (Config(..), def)
+import Text.RE.TDFA.String
 
+-- Spec
+import qualified Test.Hspec as H
+import System.IO
 
 -----------------------------------------------------------------------------
 -- Tests
 -----------------------------------------------------------------------------
 
+logFile :: FilePath
+logFile = "/tmp/test_guide.log"
```

Comment:
This only works on Unix systems. It's better to use `withTempFile` or something, see http://hackage.haskell.org/package/temporary-1.3/docs/System-IO-Temp.html#v:withTempFile